addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -306,6 +306,12 @@ var KanbanRenderer = BasicRenderer.extend({
         var ghostDefs = [];
         for (var $ghost, i = 0; i < nbDivs; i++) {
             $ghost = $('<div>').addClass('o_kanban_record o_kanban_ghost');
+            var recordView = $(fragment).find(".o_kanban_record:not(.o_kanban_ghost)");
```

Comment:
This is not good for multiple reasons:
- You create a jquery object at each turn of the loop and search for records each turn too
- The function should not have to know about kanban records
- jQuery variable names must start with `$`

-> The method must be called with a new parameter (`options` or whatever) which indicates an optional inline width for the ghosts (so it is the role of the caller to ask for the right size).

Reply:
please check the changes.